docs/spec/wake-up-welcome-message.en-US.md

```
@@ -26,7 +26,7 @@ title: 唤醒｜欢迎提示
 
 > **注：** 欢迎组件通常搭载推荐组件，可适用于不同场景。
 
-![](https://mdn.alipayobjects.com/huamei_iwk9zp/afts/img/A*6n0jSYQyAz0AAAAAAAAAAAAADgCCAQ/fmt.webp)
+![](https://mdn.alipayobjects.com/huamei_iwk9zp/afts/img/A*_MGXTpIAO8wAAAAAAAAAAAAADgCCAQ/fmt.webp)
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)